conftest.py

```diff
@@ -22,6 +22,18 @@ def pytest_addoption(parser):
         help="Optional param: version of API under test"
     )
 
+    parser.addoption(
+        "--qa_collection_user",
+        dest="qa_collection_user",
```

Comment:
Nit: This is the default dest value (even if you switch to `-` as commented above) so is onlu valuable if you want to be explicit about the destination name.